### PR TITLE
Program.cs and problem compleated

### PR DESCRIPTION
**Pull request by Fayzulloabdullayev09**

### CODE REVIEW

File: TheConversationsBot/Domain/Models/User.cs [hunk 1]
```diff
@@ -8,4 +8,5 @@ public class User
     [Description("telegram_client_id")] public long TelegramClientId { get; set; }
     [Description("phone_number")] public string PhoneNumber { get; set; }
     [Description("password")] public string Password { get; set; }
+    public long UserId { get; set; }
```
Comment (0605AbMu) [inline TheConversationsBot/Domain/Models/User.cs:11]: If UserId is Db table column then Set its Description

File: TheConversationsBot/TelegramBot/SessionManager.cs
Comment (0605AbMu) [inline TheConversationsBot/TelegramBot/SessionManager.cs:1]: I think, This is copied from another team project source. It's very bad!

**Review by 0605AbMu — COMMENTED** [review]
There are dirty codes!

File: TheConversationsBot/TelegramBot/SessionManager.cs
Comment (Fayzulloabdullayev09) [reply TheConversationsBot/TelegramBot/SessionManager.cs:1]: ok.Undersandable..

**Pull request closed without merging**